PyDSS/modes/QSTS.py

```diff
@@ -2,44 +2,37 @@
 from PyDSS.modes.abstract_solver import abstact_solver
 import math
 
-
-
-
 class QSTS(abstact_solver):
     def __init__(self, dssInstance, SimulationSettings, Logger):
         super().__init__(dssInstance, SimulationSettings, Logger)
         print("Entered QSTS mode")
         self.Settings = SimulationSettings
         self.pyLogger = Logger
-        StartDay = SimulationSettings['Project']['Start Day']
-        StartTimeMin = SimulationSettings['Project']['Start Time (min)']
-        EndTimeMin = SimulationSettings['Project']['End Time (min)']
-        sStepResolution = SimulationSettings['Project']['Step resolution (sec)']
 
-        self._Time = datetime.strptime('{} {}'.format(SimulationSettings['Project']['Start Year'],
-                                                       SimulationSettings['Project']['Start Day'] +
-                                                       SimulationSettings['Project'][
-                                                           'Date offset']
-                                                       ), '%Y %j')
-        self._Time = self._Time + timedelta(minutes=StartTimeMin)
+
+        self._Time = datetime.strptime(SimulationSettings['Project']["Start time"], "%d/%m/%Y %H:%M:%S")
         self._StartTime = self._Time
-        self._EndTime = datetime.strptime('{} {}'.format(SimulationSettings['Project']['Start Year'],
-                                                          SimulationSettings['Project']['End Day'] +
-                                                          SimulationSettings['Project'][
-                                                              'Date offset']
-                                                          ), '%Y %j')
+        self._EndTime = self._Time + timedelta(minutes=SimulationSettings['Project']["Simulation duration (min)"])
+        StartDay = (self._StartTime - datetime(self._StartTime.year, 1, 1)).days + 1
+        StartTimeMin = self._StartTime.minute
+        sStepResolution = SimulationSettings['Project']['Step resolution (sec)']
+
+        self.StartDay = (self._StartTime - datetime(self._StartTime.year, 1, 1)).days + 1
+        self.EndDay = (self._EndTime - datetime(self._EndTime.year, 1, 1)).days + 1
 
-        self._EndTime = self._EndTime + timedelta(minutes=EndTimeMin)
         self._sStepRes = sStepResolution
         self._dssIntance = dssInstance
         self._dssSolution = dssInstance.Solution
         self._dssSolution.Mode(2)
+
         self._dssSolution.Hour((StartDay - 1) * 24)
-        self._dssSolution.Seconds(StartTimeMin * 60)
+        self._dssSolution.Seconds(StartTimeMin * 60.0)
```

Comment:
This is not correct. `StartTimeMin` is relative to the hour in `StartTimeMin`. The exports will only be correct if the user specifies a start time of midnight.